var/spack/repos/builtin/packages/resolve/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Resolve(CMakePackage, CudaPackage, ROCmPackage):
+    """ReSolve is a library of GPU-resident sparse linear solvers. It contains iterative and direct
+    solvers designed to run on NVIDIA and AMD GPUs, as well as CPU devices."""
+
+    homepage = "https://github.com/ORNL/ReSolve"
+    url = "https://github.com/ORNL/ReSolve.git"
```

Comment:
```suggestion
    url = "https://github.com/ORNL/ReSolve.git"
    git = "https://github.com/ORNL/ReSolve.git"

```

Reply:
Actually 'url' is for taballs - so I guess it can be skipped - if not traballs are used..

```suggestion
    git = "https://github.com/ORNL/ReSolve.git"
```